lib/Chunk.js

```diff
@@ -396,8 +396,12 @@ class Chunk {
 		}
 		otherChunk._groups.clear();
 
-		if (this.name && otherChunk.name) {
-			if (this.name.length !== otherChunk.name.length) {
+		// Pick a new name for the chunk; prefer chunks with an entry module
+		// Otherwise, choose chunk with shortest name
+		if (this.name && otherChunk.name && !this.entryModule) {
```

Comment:
Also handle the case were both chunks have `entryModule`s

Reply:
The first is picked at the moment. Do you want to fallback to shortest name then?

Reply:
yep, need to be deteministic